test/manifolds/probability_simplex.jl

```diff
@@ -57,6 +57,8 @@ include("../utils.jl")
                 inverse_retraction_methods=[SoftmaxInverseRetraction()],
                 retraction_methods=[SoftmaxRetraction()],
                 test_inplace=true,
+                test_rand_point=true,
+                #test_rand_tvector=true, # implemented, but turned off until atol can be set locally
```

Comment:
Could you elaborate? Where is that missing? In the probability simplex or in the test suite?

You could also write a test outside this test suite: set the seed, call both rand-variants (points / tangent vectors) and test that you got a point / a vector, ,resp.